simulcast.html

```diff
@@ -0,0 +1,410 @@
+      multiple RTP encodings as defined in [[!JSEP]], the
+      <code><a>RTCRtpReceiver</a></code> may receive multiple RTP
+      encodings and the parameters retrieved via the transceiver's
+      <code>receiver.getParameters()</code> will reflect the
+      encodings negotiated.</p>
+    </section>
+    <section id="setparameters-validation">
+      <h3>setParameters</h3>
+      <p>When the <code>RTCRtpSender</code> object's <code>setParameters</code> method is called,
+      the user agent MUST run the following steps:</p>
+       <ol>
+         <li>Let <var>parameters</var> be the method's first argument.
+         </li>
+         <li>Let <var>sender</var> be the
+         <code><a>RTCRtpSender</a></code> object on which
+         <code>setParameters</code> is invoked.</li>
+         <li>Let <var>transceiver</var> be the
+         <code><a>RTCRtpTransceiver</a></code> object associated
+         with <var>sender</var> (i.e. <var>sender</var> is
+         <var>transceiver</var>'s <a>[[\Sender]]</a>).</li>
+         <li>Let <var>N</var> be the number of
+         <code><a>RTCRtpEncodingParameters</a></code> stored in
+         <var>sender</var>'s internal <a>[[\SendEncodings]]</a>
+         slot.</li>
+         <li>If any of the following conditions are met, return a promise
+         <a>rejected</a> with a newly
+         <a data-link-for="exception" data-lt="create">created</a>
+         <code>InvalidModificationError</code>:
+         <ol>
+            <li><code><var>parameters</var>.encodings.length</code>
+            is different from <var>N</var>.</li>
+            <li><code><var>parameters</var>.encodings</code> has been
+            re-ordered.</li>
+            <li>Any parameter in <var>parameters</var> is marked as a
+            <dfn>Read-only parameter</dfn> and has a value that is different
+            from the corresponding parameter value in <var>sender</var>'s
+            <a>[[\LastReturnedParameters]]</a> internal slot. Note that
+            this also applies to <var>transactionId</var>.</li>
+         </ol>
+         </li>
+         <li>
+           <p>For each value of <var>i</var> from 0 to the number of encodings,
+           check whether
+           <code><var>parameters</var>.encodings[<var>i</var>].codecPayloadType</code>
+           (if set) corresponds to a value of
+           <code><var>parameters</var>.codecs[<var>j</var>].payloadType</code> where
+           <var>j</var> goes from 0 to the number of codecs. If there is no
+           correspondence, or if the MIME subtype portion of
+           <code><var>parameters</var>.codecs[<var>j</var>].mimeType</code> is equal to
+           "red", "cn", "telephone-event", "rtx" or a forward error correction
+           codec ("ulpfec" [[RFC5109]] or "flexfec" [[FLEXFEC]]), reject <var>p</var> with
+           a newly <a data-link-for="exception" data-lt="create">created</a>
+           <code>InvalidAccessError</code>.</p>
+         </li>
```

Comment:
Are there steps missing from the current setParameters validation? scaleResolutionDownBy? maxFramerate?

Reply:
I am sure there are *many* steps missing.  Once we have a separate specification we can file issues to address this.